### PR TITLE
Our-technology-partnerships-section

### PR DESCRIPTION
**Pull request by GBouffard**
## Description - [526](https://trello.com/c/RI5Mps0H/526-our-technology-partnerships-section)

put a short summary of what you did here:
- [x] Add to existing graphql query in open-source.js (on open-source-page branch)
- [x] Create Partners section in /OpenSource folder
Also added:
- [x] open source page directly accessible from Services Dropdown menu (as it will be in the future)
- [x] graphQL query retrieve data from Contenteful (for the whole openSource page)
- [x] Added Partnerships section into open page (above small paragraph to see it appear properly for now)

nb: the resizing of the logos is a bit tricky as they have different length/width so I used a container to make sure they will be restricted and kind of resize the same way.

## Review template

- [ ] checked that this PR resolves the spec provided from trello / this description
- [ ] ensured that this PR does not introduce any obvious bugs

### CODE REVIEW

File: src/components/OpenSource/Partnerships.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      themeVariation="dark"
+    />
+  </Col>
+)
+
+const PartnershipsSection = ({ title, partners }) => (
+  <BlueBackground>
+    <Grid>
+      <Row>
+        <StyledSectionTitle width={[1 / 2]}>
```
Comment (omonk) [inline src/components/OpenSource/Partnerships.js]: Will this go to full width at mobile size?

File: src/components/OpenSource/Partnerships.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+          <SectionTitle reverse>{title}</SectionTitle>
+        </StyledSectionTitle>
+      </Row>
+      <StyledPartnersRow>
+        {partners.map(partner => PartnerCol(partner))}
```
Comment (omonk) [inline src/components/OpenSource/Partnerships.js]: `{partners && partners.length && partners.map(....`

File: src/components/OpenSource/Partnerships.js (hunk 1 above)
Comment (GBouffard) [reply src/components/OpenSource/Partnerships.js]: Nope. Need to change. Good point! 👍

File: src/components/OpenSource/Partnerships.js (hunk 2 above)
Comment (GBouffard) [reply src/components/OpenSource/Partnerships.js]: Yep. Safety net to add 👍

**Pull request merged**